src/DrawTogether.AppHost/Configuration.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace DrawTogether.AppHost;
+
+public class DrawTogetherConfiguration
+{
+    /// <summary>
+    /// Whether to use persistent volumes for our database.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to false so we don't accidentally persist data during testing.
+    /// </remarks>
+    public bool UseVolumes { get; set; } = false;
```

Comment:
We configure `UseVolumes` to be disabled by default during testing, but it's set to `true` inside `appSettings.json`